spec/features/user_searches_stations_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+feature "visitor searches for station", %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.